### ISSUE TITLE
SASS paths and output style

### ISSUE THREAD

**Issue by chrisfinazzo** [thoughtbot/bourbon#621]
Although I've only been trying out Bourbon for a few days, I like what I've seen so far. However, when it comes to SASS and building the site (I'm using Jekyll, btw) I think I'm missing something...

In Compass, resource paths and style formatting gets declared in a `config.rb` file. Set it once, and you really don't need to look at it again. Really simple.

How does Bourbon deal with these options? Having to run `sass --watch` inside of each directory seems clunky. I'd much rather make a small bash script that chains the commands together. With the path specified, it should just be able to run from the project root, no?

Ideally, I would like to have 1 `.scss` file which imports all partials and outputs a compressed format with all line breaks and extra spaces removed.

I know there are [several][] [projects][] which implement Bourbon and Neat as Compass extensions, but that looks kind of weird and I'm not sold on it working long-term...

[several]: https://github.com/jedfoster/bourbon-compass

[projects]: https://github.com/jedfoster/neat-compass

**Comment by tysongach** [comment]
Bourbon is just a library of Sass partials. Just import it in your main Sass manifest and you can start using it.

Jekyll comes with [built-in support](http://jekyllrb.com/docs/assets) for Sass, so when you run `jekyll serve` it should be continually compiling Sass for you. It has some options for output style, too.

**Comment by chrisfinazzo** [comment]
@tysongach,

Thanks for the quick reply, it figures that Jekyll has configuration options for this (and I've never seen them :smile:. Although I've contributed to Jekyll (and its docs) in the past, this is one area I didn't bother with. Now that I'm using SASS in more projects, it's definitely worth a read.

I first started trying to get Bourbon up and running yesterday and kept running into conflicts between the file structure that Bourbon expects and what you get out of `jekyll new`, hence the confusion.
